services/workflows-service/src/filter/filter.repository.ts

```diff
@@ -27,10 +27,13 @@ export class FilterRepository {
 
   async findById(id: string, args: Prisma.FilterFindFirstArgs, projectIds: TProjectIds) {
     return await this.prisma.filter.findFirst(
-      this.scopeService.scopeFindFirst({
-        ...args,
-        where: { ...args?.where, id: id },
-      }),
+      this.scopeService.scopeFindFirst(
+        {
+          ...args,
+          where: { ...args?.where, id: id },
+        },
+        projectIds,
```

Comment:
Boyscouting. We didn't use `projectIds` for some reason, even though it's not an unscoped function